pymobiledevice3/cli/profile.py

```diff
@@ -36,6 +36,17 @@ def profile_install(lockdown: LockdownClient, profiles):
         logger.info(f'installing {profile.name}')
         service.install_profile(profile.read())
 
+@profile_group.command('install-silent', cls=Command)
+@click.option('--keystore', type=click.File('rb'), required=True, help="A PKCS#12 keystore containing the certificate and private key which can supervise the device.")
+@click.option('--keystore-password', prompt=True, required=True, hide_input=True, help="The password for the PKCS#12 keystore.")
+@click.argument('profiles', nargs=-1, type=click.File('rb'))
+def profile_install(lockdown: LockdownClient, profiles, keystore, keystore_password):
```

Comment:
Either:
- rename to `profile_install_silent`
- or add a `--silent` flag to previous command (`profile install`)